src/components/OSCALControlModification.js

```diff
@@ -42,13 +46,33 @@ function getAlterAddsOrRemovesDisplay(addsElements, addsLabel, controlPartId) {
         Name: {item.name}, Value: {item.value}
       </Typography>
     ));
+  let removedTypographies = null;
+  if (addsRemovesLabel === "Removes ") {
+    removedTypographies = addsElements
+      .flatMap((element) => element ?? [])
+      .map((item) => (
+        <Typography
+          color="textSecondary"
+          paragraph
+          variant="body1"
+          key={item.id}
+          style={{ textDecorationLine: "line-through" }}
+        >
+          Attribute: {Object.keys.length > 0 ? Object.keys(item)[0] : ""},
+          Value: {Object.values.length > 0 ? Object.values(item)[0] : ""}
```

Comment:
As far as design goes, I feel like this may not be the best strategy. What you are doing now is just displaying the `json` reference on what should be removed with a strike through. Based on my understanding, a `remove` element is a reference to the component to remove. 

> Perhaps a simple strike through font variant of the control element removed would be a sufficient visual indicator

I'd assumed based on Ray's comment you would use that remove reference in the `json` to find the component to remove. I'm not sure the best design from there and maybe this should have been discussed earlier. Some intial ideas could be:
- You may want to put a strike though on the element being referenced if it is currently being displayed. 
- Another potential option would be to list the information of the element being referenced with a strike through.

Reply:
I think this is a decent start overall to the design, but yes, I feel like the strikethrough isn't in the intended place.

> You may want to put a strike though on the element being referenced if it is currently being displayed.

I favor this approach most. I believe this is what Ray was intending in the issue but didn't explicitly say. I would still include the "Removes" section in the "Modifications" box (the json reference), but have the removed text be clear here (no strikethrough).

Reply:
> I would still include the "Removes" section in the "Modifications" box (the json reference), but have the removed text be clear here (no strikethrough).

I think something to consider if we do include that, is all `by-___` [properties](https://pages.nist.gov/OSCAL/reference/latest/complete/json-reference/#/profile/modify/alters/removes/by-name) can be used to identify multiple items to remove. I'd argue it is more useful to list everything we remove rather than just listing the criteria we are using to remove things. 

We even could display the criteria and then all items removed underneath it. Something like:

---

## Modifications

### Removes

#### Attribute: `by-id` Value: `ac-2_smt.c`

- item 1
- item 2

---

Reply:
I revised the code this a screenshot the new implementation
<img width="1025" alt="image" src="https://user-images.githubusercontent.com/59936247/227515049-3b35170c-3868-4c1f-9333-7f07adede336.png">